### DISCUSSION TITLE
Group Assignment #2 - Memo 📋

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#38] [Group Assignment]
## Due 7/10

Have one person comment on this post to submit the link to your [group assignment #2](https://albertkun.github.io/24SU-ASIAAM-191A/assignments/week2/group_assignment/) memo with your **group name** in this thread.

### For example

Awesome Group - [https://docs.google.com/document/d/1OWUUxebPNsqw4K97gznvTkGLEr6QDAbXWqX2BGvernI/copy](https://docs.google.com/document/d/1OWUUxebPNsqw4K97gznvTkGLEr6QDAbXWqX2BGvernI/copy)

**Comment by sjeyakum** [comment]
H2Opinions - https://docs.google.com/document/d/1b9X6kJnn1slBXMDVMdSfmPFM8iM45qzTEgy_D1X0OvE/edit?usp=sharing

**Comment by stinest** [comment]
Young Voices of Solidarity, In Support of Resistance - https://docs.google.com/document/d/1Pvp0Lycq1uSwPRjI9X7VpGJz8C_6isw6jjHTKnTOQ3E/edit

**Comment by nesslee123** [comment]
Is UCLA Really Accessible? - https://docs.google.com/document/d/15u_kiQSaHCSrXbj5ztg-TvegGCrD4kP5x1LTQI9BvoA/edit?usp=sharing

**Comment by chrissanrow** [comment]
LGBTQInternational - https://docs.google.com/document/d/17N6CfHZ5D9SkhxuwUiCvE-xjefvPTlcsejIajCLfCD4/edit?usp=sharing

**Comment by kianaptran** [comment]
Group ATE (UCLA Food Accessibility) - https://docs.google.com/document/d/1oJ6rpDbYNWncaZ6SWXQLljPqgCmlFJ1Y2EZXVGOVLAs/edit?usp=sharing

**Comment by Keiya-Wada** [comment]
Japanese - https://docs.google.com/document/d/1DAR8vedLzE7lWqSPMEQud2DMmEt61dvV8MwKJ9__nTk/edit?usp=sharing

**Comment by caitlynn-hauw** [comment]
Post-incarceration - https://docs.google.com/document/d/16jq3E8JehakP4f8Bx7aVSKWRN1-oPGM15rPbsb8WiH0/edit?usp=sharing
